<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>17</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar_8">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>配方1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>配方2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>配方3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
